lib/season_stats.rb

```diff
@@ -9,24 +8,18 @@
 
 class Season
   include Calculable
-  
-
-  # season, game_id      'games_fixtures'
-  # game_id, team_id, tackles   'games_teams_fixtures'
-  # team_id, franchiseId 'teams.csv'
 attr_reader :year, :teams, :games, :game_teams, :searched_season
-  def initialize(year)
-    #Add database arguments later
+  def initialize(year,teams_database, games_database, game_teams_database)
```

Comment:
remove year from initialize, make it an argument for each method or or searched_season